tests/TestCase/Routing/RouterTest.php

```diff
@@ -3116,6 +3117,125 @@ public function testConnectShortStringSyntax()
         $this->assertEquals($result, $expected);
     }
 
+    /**
+     * test url() with short string syntax
+     *
+     * @return void
+     */
+    public function testUrlGenerationInShortStringSyntax()
+    {
+        Router::connect('/articles', 'Articles::index');
+        Router::connect('/articles/view/*', 'Articles::view');
+        Router::connect('/article/{slug}', 'Articles::read');
+        Router::connect('/admin/articles', 'Admin/Articles::index');
+        Router::connect('/cms/articles', 'Cms.Articles::index');
+        Router::connect('/cms/admin/articles', 'Cms.Admin/Articles::index');
+
+        $result = Router::url('Articles::index');
+        $expected = '/articles';
+        $this->assertEquals($result, $expected);
+
+        $result = Router::url('Articles::view', [3]);
+        $expected = '/articles/view/3';
+        $this->assertEquals($result, $expected);
+
+        $result = Router::url('Articles::read', ['slug' => 'title']);
+        $expected = '/article/title';
+        $this->assertEquals($result, $expected);
+
+        $result = Router::url('Admin/Articles::index');
```

Comment:
I wonder if we should also support bare strings for named routes 🤔 . It should be a separate set of changes after these ones.